doc/src/manual/faq.md

```diff
@@ -645,6 +645,13 @@ as nothing but rather a tuple of zero values.
 The empty (or "bottom") type, written as `Union{}` (an empty union type), is a type with
 no values and no subtypes (except itself). You will generally not need to use this type.
 
+
+### How do I check if a file is directly executed by Julia?
```

Comment:
A `.jl` file is always executed by Julia so I find this working confusing. How about:

> How do I check if the current `.jl` file is being run as the main script?